doc/developers/maintainer.rst

```diff
@@ -1,14 +1,27 @@
 Maintainer / core-developer information
 ========================================
 
+A release is either a major release, or a minor bug fix release. Our convention
+is that we release one or more release candidates (0.XXrcN) before releasing
+the final distributions. Creating the first RC involves creating the release
+branch (0.99.X for instance) directly on the main repo. The rest of the changes
+should be done as a PR to the master branch, if it's possible, or done through
+a PR to the release branch, for the other maintainers to review before the
+release. This is further explained under
+:ref:`Preparing a release PR<Preparing a release PR>`. We follow the
```

Comment:
... me asking for forgiveness for the sphinx warning... this time I've checked
 ```suggestion
`Preparing a release PR<#Preparing-a-release-PR>`_. We follow the
```

Reply:
haha, thanks for the fix.